lib/chai/core/assertions.js

```diff
@@ -1285,130 +1286,104 @@ module.exports = function (chai, _) {
    * @name throw
    * @alias throws
    * @alias Throw
-   * @param {ErrorConstructor} constructor
-   * @param {String|RegExp} expected error message
+   * @param {Error|ErrorConstructor} errorLike
+   * @param {String|RegExp} errMatcher error message
    * @param {String} message _optional_
    * @see https://developer.mozilla.org/en/JavaScript/Reference/Global_Objects/Error#Error_types
    * @returns error for chaining (null if no error)
    * @namespace BDD
    * @api public
    */
 
-  function assertThrows (constructor, errMsg, msg) {
+  function assertThrows (errorLike, errMatcher, msg) {
```

Comment:
I'm not crazy about the terms `errorLike` and `errMatcher`, partially because one is "error" and the other "err", and partially because I feel some context is lost with "errMatcher".

What do you think about `errLike` and `errMsgMatcher`?